after/plugin/compe_ultisnips.vim

```diff
@@ -0,0 +1,9 @@
+if exists('g:loaded_compe_ultisnips')
+  finish
+endif
+let g:loaded_compe_ultisnips = v:true
+
+if exists('g:loaded_compe')
```

Comment:
Could you check ultisnips installed or not?

Reply:
Yeah! I would include the check using ``match`` function of vimscript